### ISSUE TITLE
Version 0.1.0

### ISSUE THREAD

**Issue by lukebarnard1** [lukebarnard1/matrix-redux-wrap#2]
This version will be a MVP to provide the minimal functionality required to make Redux-based app. 

Requirements for this are:
 - [x] supporting simple authentication, login, logout functionality;
 - [x] supporting simple registration functionality
 - ~~being able to persist store state across browser sessions (e.g. via localStorage);~~ this is out-of-scope, see redux-storage or something similar
 - [x] being able to sync with a matrix server;
 - [x] exposing current sync state (for online/offline indicator);
 - [x] exposing essential matrix state via matrixReduce, namely:
   - [x] the rooms known to a user: 
        - [x] room name;
        - [x] the user's membership;
        - [x] members; 
        - [x] most recent events (a single live timeline);
             - [x] with scrollback;

**Comment by lukebarnard1** [comment]
Once all of this is finished, most  (if not all) of the state management in [journal](https://github.com/lukebarnard1/journal) can be replaced with matrix-redux-wrap, see https://github.com/lukebarnard1/journal/issues/62

**Comment by lukebarnard1** [comment]
With registration and login examples now committed, I think it could be quite tractable now to go and move journal onto the new storage. 

It's true that it might be worth moving some of the more generic `doLoginAndSync`-style functions in the examples into the library itself, but this can be done later. Copying them into journal will be a good way of finding out whether they are suited to being library functions.

**Comment by lukebarnard1** [comment]
Oh I guess I should release this now :)
